### DISCUSSION TITLE
Configuration Error - "'NoneType' object has no attribute 'find_all'"

### DISCUSSION THREAD

**Discussion by rbensonx** [mampfes/hacs_waste_collection_schedule#2552] [Q&A]
removed the old integration, cleaned out configuration and tried a reinstall. 

I'm getting the error...

The source returned an invalid response: "'NoneType' object has no attribute 'find_all'". Please check the provided arguments and try again.

Any ideas?

New Zealand, Auckland Council. Version 2.1.0

**Comment by 5ila5** [comment]
The website seem to return a error page now if there is no User-Agent header.

Fixed this in #2558

This will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by rbensonx** [comment]
Awesome. Much appreciated!
